ablySpec/RealtimeClientChannel.swift

```diff
@@ -609,6 +609,62 @@ class RealtimeClientChannel: QuickSpec {
 
             }
 
+            // RTL8
+            context("unsubscribe") {
+
+                // RTL8a
+                it("with no arguments unsubscribes the provided listener to all messages if subscribed") {
+                    let client = ARTRealtime(options: AblyTests.commonAppSetup())
+                    defer { client.close() }
+
+                    let channel = client.channels.get("test")
+
+                    waitUntil(timeout: testTimeout) { done in
+                        channel.subscribe { message in
+                            expect(message.data as? String).to(equal("message"))
+                            done()
+                        }
+                        channel.publish(nil, data: "message")
+                    }
+
+                    channel.unsubscribe()
+
+                    channel.publish(nil, data: "message")
+
+                    // If `unsubscribe` fails then the test suite will raise "Done closure's was called multiple times."
+                    waitUntil(timeout: testTimeout) { done in
+                        delay(5.0) { done() }
```

Comment:
Could you please make this delay much smaller and move it to the callback of the second `channel.publish`? The test suite is already way too slow as it is...